app/forms/network-interface-create.tsx

```diff
@@ -23,7 +23,7 @@ import { useForm, useProjectSelector } from 'app/hooks'
 const defaultValues: InstanceNetworkInterfaceCreate = {
   name: '',
   description: '',
-  ip: '',
+  ip: undefined,
```

Comment:
I believe this will only work with the initial value, but not if you type something in and then delete it.

Reply:
Yep, you're right.